### ISSUE TITLE
the table gender_names is missing

### ISSUE THREAD

**Issue by AndryCola** [PokeAPI/pokeapi#964]
i noticed that exists the table "genders", but the table with the names is missing.
can you add it please?

**Comment by SKCwillie** [comment]
I'm not sure what you mean by this?

The genders table includes the id, 1-3 with the names, female, male, genderless. Are you looking for something other than that?

**Comment by AndryCola** [comment]
yes, i am. i was looking for something like id, name, local_language_id for the translation of genders

**Comment by Naramsim** [comment]
Hi, if you could help us add it and implement the logic we would be really happy!
